python/src/FunctionalChaosResult_doc.i.in

```diff
@@ -212,3 +223,186 @@ coefficientsHistory : 2-d sequence of float
     The coefficients values selection history
     Must be of same size as indicesHistory."
 
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::FunctionalChaosResult::getConditionalExpectation
+"Get the conditional expectation of the expansion given one vector input.
+
+This method returns the functional chaos result corresponding to the conditional 
+expectation of the output given an input vector.
+Indeed, the conditional expectation of a polynomial chaos expansion is, again, 
+a polynomial chaos expansion.
+Let :math:`\physicalInputRandomVector \in \Rset^{\physicalInputDimension}` be the input and let :math:`\vect{u} \subseteq \{1, ..., \physicalInputDimension\}`
+be a set of marginal indices. 
+Let :math:`\physicalInputRandomVector_{\vect{u}} \in \Rset^{|\vect{u}|}` be the vector 
+corresponding to the group of input variables where :math:`|\vect{u}| = \operatorname{card}(\vect{u})`
+is the number of input variables in the group.
+Let :math:`\surrogateModel(\physicalInputRandomVector)` be the polynomial chaos
+expansion of the physical model :math:`\model`.
+This function returns the functional chaos expansion of:
+
+.. math::
+
+    \surrogateModel_{\vect{u}}\left(\physicalInputObservation_{\vect{u}}\right) 
+    = \Expect{\surrogateModel(\physicalInputRandomVector) | \physicalInputRandomVector_{\vect{u}} = \physicalInputObservation_{\vect{u}}}
+
+for any :math:`\physicalInputObservation_{\vect{u}} \in \Rset^{|\vect{u}|}`.
+
+**Mathematical analysis**
```

Comment:
> > You are using many notations which are not defined here (but are defined in the theory I believe). Maybe this whole section could be move to a theory page where the notations are defined?



> That is true: I have to do this.

OK, just putting this comment to keep this topic afloat in the sea of GitHub comments.

Reply:
Actually, it is difficult to cover the whole PCE theory within a single help page, isn'it? This is why in c88015fd6b7c054eb55624c4635df596adc22e60, I inserted a link to the theory help page and a link to the associated example. I think that increasing the current size of the theory help page of the PCE in theory/meta_modeling/functional_chaos.rst may complicate a help page which is already relatively complicated.

Is this ok?